samples/bluetooth/peripheral_hids_mouse/child_image/hci_rpmsg.conf

```diff
@@ -6,3 +6,4 @@
 
 CONFIG_BT_MAX_CONN=2
 CONFIG_BT_CENTRAL=n
+CONFIG_BT_CTLR_PRIVACY=y
```

Comment:
Is it needed to enable `CONFIG_BT_CTLR_PRIVACY`?
It should be enabled by default because `CONFIG_BT_HCI_RAW` is configured in the  hci_rpmsg sample.

Reply:
Correct, removed the setting.